docs/docs/configuration/camera_specific.md

```diff
@@ -231,3 +231,37 @@ ffmpeg:
 ### TP-Link VIGI Cameras
 
 TP-Link VIGI cameras need some adjustments to the main stream settings on the camera itself to avoid issues. The stream needs to be configured as `H264` with `Smart Coding` set to `off`. Without these settings you may have problems when trying to watch recorded footage. For example Firefox will stop playback after a few seconds and show the following error message: `The media playback was aborted due to a corruption problem or because the media used features your browser did not support.`.
+
+## USB Cameras (aka Webcams)
+
+In some scenarios one may desire using a USB Camera (i.e. webcam) to use with frigate. You can first check to see what the USB Camera device path is along with what its supported formats and dimensions are using `v4l-utils`, which you can install on a Debian GNU/Linux system with `sudo apt install v4l-utils`. You may then run `v4l2-ctl --list-devices` to get a listing of locally-connected cameras available. Assuming you have a camera connected to `/dev/video0`:
+
+- Outside of Frigate, run `ffmpeg -f v4l2 -list_formats all -i /dev/video0` to get an idea of what formats and resolutions the USB Camera supports.
+
+- If using Frigate in a container (e.g. Docker on TrueNAS), ensure you have USB Passthrough support enabled, along with a specific Host Device (`/dev/video0`) + Container Device (`/dev/video0`) listed.
+
+- In your Frigate Configuration File, list the USB camera with ffmpeg arguments that:
+  - input args: reference the device (e.g. `/dev/video0`) and v4l2 (e.g. `-f v4l2`)
```

Comment:
```suggestion
  - input args: reference v4l2 (e.g. `-f v4l2`)
```

Reply:
The `path` is what should reference the device, not the `input_args`. Your example below is correct, but this wording here is inconsistent.